client/src/App/App.js

```diff
@@ -107,12 +103,12 @@ function App() {
 					setModalForInfoIsOpen(true);
 					return;
 				}
-				if (body.sent === true) {
+				if (body.sent) {
 					setInfoText("Your letter was sent.");
 					setModalForInfoIsOpen(true);
 					return;
 				}
-				if (body.sent === false) {
+				if (!body.sent) {
```

Comment:
rate limits has status code from http spec, use it pls

Reply:
fixed by using 429 status code